plugins/woocommerce-admin/client/customize-store/design-with-ai/services.ts

```diff
@@ -220,6 +248,11 @@ export const updateStorePatterns = async (
 			},
 		} );
 
+		if ( ! images.length ) {
```

Comment:
Hey @albarin 👋  I just tested the trunk branch on a JN site and couldn't generate content/images with AI.

It seems that `/wc/private/ai/images` endpoint returned a different format than expected. Could you take a look?

![Screenshot 2024-01-04 at 21 02 17](https://github.com/woocommerce/woocommerce/assets/4344253/61d6cd1c-829c-4f4f-b974-2a062795dea0)

Reply:
That's weird, I'll have a look, it was testing fine in WooExpress 🤔

Reply:
I have opened a PR to address this, thanks for spotting this https://github.com/woocommerce/woocommerce/pull/43285